app/views/admin/payroll_runs/index.html.erb

```diff
@@ -7,15 +7,11 @@
   <div class="govuk-grid-column-two-thirds">
     <p class="govuk-body">Make a payroll file containing all the relevant information for approved claims to be paid by a payroll provider.</p>
 
-    <div class="govuk-warning-text">
-      <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
-      <strong class="govuk-warning-text__text">
-        <span class="govuk-warning-text__assistive">Warning</span>
-        Only one payroll file should be generated and submitted per calendar month
-      </strong>
+    <div class="govuk-inset-text">
+      The next payroll file should be uploaded by <%= l(next_payroll_file_upload_date) %>
```

Comment:
The word "Upload" also appears in the payroll history, maybe we should use different words here to not confuse sending the file to Cantium with uploading the payment confirmation report?

```suggestion
      The next payroll file should be sent to Cantium by <%= l(next_payroll_file_upload_date) %>
```

Reply:
Also, if the file has been generated for the month, does it still make sense to show this message?

Reply:
If the button is showing on the first of the month I may be tempted to (or accidentally) press it, not knowing that would have an impact on that months payroll run. 

Perhaps we should show the button at a sensible date __before__ the cantium deadline? If that was the behaviour, it would be helpful to show the user why there is no button (it's not time yet) and when there will be a button to press (something like two working days before the canitum deadline, which I am believe is the third friday of the month) or do we feel like that is too restrictive?

Reply:
I think this is over-thinking it a bit. Pressing the button early would only have an impact if the user then also clicked the "Create payroll file" button on the subsequent page and actually created the payroll run. That screen also shows the number of approved claims and the value of those claims, which should give a strong signal that it might be too early.

Reply:
Okay, well I've updated it to calculate the third friday of the next month, which makes it this:

![Screenshot_2019-11-13 Claim additional payments for teaching – GOV UK](https://user-images.githubusercontent.com/480578/68784722-e8155680-0634-11ea-8a78-3e2727ff707b.png)

There was a comment on the card about showing something around the schedule. Do we want to keep it or loose it?

Reply:
Looks like it might be the second Friday:

https://drive.google.com/file/d/0B4DnBBYceUWQd2xXZDlxVjI4RC1fS2xzMXEwWGJmWHhKS19R/view

Reply:
Ah yeah, it looks slightly more subtle than that even. It looks like it's the end of the second full week of the month. So some month's it's the third Friday, and some the second. Perhaps we should speak to Cantium to get this nailed down?

Reply:
Sounds good. I just need to get the schedule clear in my head so I know what I am asking.

Reply:
Let me know off I can help with that.

Reply:
I spoke with Chris and he is pretty clear:

- the 'pay day Friday' for claimants is the second to last friday of the month
- Cantium get five full workings days to perform the payroll
- the file is due to Cantium the Friday before the 'pay day Friday'
- I am not worrying about bank holidays as I feel like service operators can deal with those and things could change by then